include/RAJA/policy/openmp_target/forall.hpp

```diff
@@ -60,9 +60,10 @@ RAJA_INLINE void forall_impl(const omp_target_parallel_for_exec<ThreadsPerTeam>&
   }
 
 // thread_limit(tperteam) unused due to XL seg fault (when tperteam != distance)
+  auto i = distance_it;
 #pragma omp target teams distribute parallel for num_teams(numteams) \
-    schedule(static, 1) map(to : body)
-  for (decltype(distance_it) i = 0; i < distance_it; ++i) {
+    schedule(static, 1) firstprivate(body,begin_it)
```

Comment:
If this is needed, how did it work before?

Reply:
The both body and begin_it were being mapped, which causes warnings on current clang while firstprivate does not.  Either should be technically correct, this should be slightly faster and cause less warnings.